CHAID/CHAID.py

```diff
@@ -358,16 +357,23 @@ def generate_best_split(self, ind, dep):
         """ internal method to generate the best split """
         split = CHAIDSplit(None, None, None, 1)
         relative_split_threshold = 1 - self.split_threshold
+        # We add one to the dependent array as when an answer hasn't been given
```

Comment:
Yeah, well. I think in cases like this where you can come back and be like "Why +1" its best to have some indication of why, either a comment or a well named constant. Here a comment seemed to make most sense